exonum-java-binding/core/pom.xml

```diff
@@ -209,6 +209,24 @@
             </goals>
           </execution>
 
+          <execution>
+            <id>build-ejb-app</id>
+            <phase>package</phase>
+            <configuration>
+              <arguments>
+                <argument>+${rust.compiler.version}</argument>
+                <argument>build</argument>
+                <argument>--package</argument>
```

Comment:
Funny fact: cargo has a short version of this flag, `-p` - but it's impossible to use it here due to bug in Maven